accelerator/errors.h

```diff
@@ -79,7 +83,11 @@ typedef enum {
   /**< JSON key not found */
   SC_CCLIENT_JSON_PARSE = 0x08 | SC_MODULE_CCLIENT | SC_SEVERITY_MAJOR,
   /**< json parsing error, might the wrong format */
-
+  SC_CCLIENT_FLEX_TRITS = 0x09 | SC_MODULE_CCLIENT | SC_SEVERITY_MAJOR,
+  /**< Flex trits converting error */
+  SC_CCLIENT_JSON_CREATE = 0x0A | SC_MODULE_CCLIENT | SC_SEVERITY_MAJOR,
```

Comment:
Index seems skip 0x02. Could you help correct it?

Reply:
I will fix this in the other PR

Reply:
I will fix this in the other PR